index.html

```diff
@@ -8,16 +8,16 @@
     <link rel="stylesheet" href="styles.css" />
 </head>
 <body>
-    <nav id="navbar">
+    <div class="navbar">
         <ul id="navlist">
             <li><a href="#welcome-section">home</a></li>
             <li><a href="#projects">projects</a></li>
         </ul>
-    </nav>

```

Comment:
Pls make sure you use nav instead of the div element, makes web pages more informative and adaptable, allowing browsers and search engines to better interpret content.